### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by aneallaryea100**
# Portfolio: validate contact form
* The content of the email field has to be in lower case.
* When the user submits the form, you check if the email is in lower case.
* If the validation is OK, the form is sent.
* If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -13,3 +13,131 @@ close.addEventListener('click', () => {
...
+form.addEventListener('submit', (element) => {
+  element.preventDefault();
+  const emailValue = email.value.trim();
+  const pat = /^[a-z]+$/;
+
+  if (pat.test(emailValue) === false) {
+    errorMessage.innerText = 'Email must be lowercase***';
+  } else {
+    form.submit();
+  }
```
Comment (Ismailco) [inline script.js]: - [ ] The form is showing the error even when the email is lowercase, kindly fix this so your form work's as expected

Screenshot_20220623_180804

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi Team,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (aneallaryea100) [reply script.js]: Form now accepts only lowercase emails.

**Review by Wes-Isaac — APPROVED** [review]
Hi @aneallaryea100, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters are all green ✔️.
- Project works as expected 🚀.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @aneallaryea100  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**